thunder/core/proxies.py

```diff
@@ -626,7 +626,22 @@ def _elementwise_unary_helper(a, name, fn):
         except Exception as e:
             return fn(vala)
 
-        return method(a)
+        if method is None:
```

Comment:
We should delete these implicit NumberProxy to number translations in the future, right?

Reply:
Marking as resolved and opened #428 to track